lms/js_config_types.py

```diff
@@ -172,3 +172,12 @@ class DashboardConfig(TypedDict):
     """Organization data for dashboard access scoped to one organization. For staff members only."""
 
     routes: DashboardRoutes
+
+
+class EmailPreferences(TypedDict):
+    is_instructor: bool
+    selectedDays: dict[str, bool]
+
+    mention_emails_feature_enabled: bool
+    mention_emails_subscribed: bool
+    flashMessage: str | None
```

Comment:
The mixture of pascalCase and snake_case here is for backwards compatibility?

Reply:
Yeah the rest of the fields on this file (that we created for dashboards) all use snake case.

I reckon then should use snake case for new fields but the existing ones are on camelCase

Reply:
> I reckon then should use snake case for new fields but the existing ones are on camelCase

Yep, I also think we should use snake_case going forward. It's unfortunate we used camelCase in some old props, but probably not the end of the world.